app/assets/stylesheets/dance_comments.scss

```diff
@@ -0,0 +1,22 @@
+#comment_box {
+  border: 3px solid #000000;
```

Comment:
This rule makes sense, especially if you are repeating colors throughout the app to give it a theme.
```
// colors.scss
$color- black: #00000

// other-file.scss
@import 'colors';

.thing {
 color: $color-black;
}
```